pages/about.vue

```diff
@@ -24,6 +24,10 @@
         <h2>{{ $t("about.supportQ") }}</h2>
         {{ $t("about.supportA") }}
       </div>
+      <div>
+        <h2>Who developed this site?</h2>
```

Comment:
If there's time, can we use i18n? (But it would be best to get this in by tomorrow morning so it can come later)

Reply:
Yeah, exactly why I didn't do that yet. I'll merge this first, then make immediate PRs to get the key in.